gui improvements

torusrxxx opened the pull request:
* 9 more shortcut configurations (fix #1260 #1255)
* Follow in memory map in stack and modules view(fix #1264). Fixed "Follow in memory map" in stack view to follow the content of the stack in the memory map, instead of the memory page of the stack before.
*  Search for various things in the current function. Due to #1270, this submenu is disabled currently.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/x64dbg/x64dbg/1272)
<!-- Reviewable:end -->

mrexodia commented on src/gui/Src/Gui/CPUStack.cpp:
```diff
@@ -1005,7 +1005,7 @@ void CPUStack::dbgStateChangedSlot(DBGSTATE state)
 void CPUStack::followInMemoryMapSlot()
 {
-    DbgCmdExec(QString("memmapdump %1").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());
+    DbgCmdExec(QString("memmapdump [%1]").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());
```
I think this behavior is better, but it should then replace `Follow in Memory Map` with `Follow DWORD in Memory Map` and also check if that DWORD is a valid memory address before adding it to the menu.

torusrxxx commented:
* Added Variables View (varlist)
* Use the same shortcut for graph refresh as in other views
* Changed to "Follow DWORD in Memory Map"
* Show size of stack call frame in call stack view

mrexodia commented on src/gui/Src/Gui/MainWindow.cpp:
```diff
@@ -402,7 +403,7 @@ void MainWindow::setupLanguagesMenu()
     QLocale enUS(QLocale::English, QLocale::UnitedStates);
     QString wCurrentLocale(currentLocale);
-    QAction* action_enUS = new QAction(QString("[%1] %2 - %3").arg(enUS.name()).arg(enUS.nativeLanguageName()).arg(enUS.nativeCountryName()), languageMenu);
+    QAction* action_enUS = new QAction(QString("[%1]%2 - %3").arg(enUS.name()).arg(enUS.nativeLanguageName()).arg(enUS.nativeCountryName()), languageMenu);
```
I changed this previously it's more natural to have a space there.

torusrxxx replied:
It is inconsistent with the next string

torusrxxx replied:
So I changed it.
